dinky-metadata/dinky-metadata-postgresql/src/main/java/org/dinky/metadata/convert/PostgreSqlTypeConvert.java

```diff
@@ -57,4 +62,14 @@ public PostgreSqlTypeConvert() {
         register("jsonb", ColumnType.STRING);
         register("json", ColumnType.STRING);
     }
+
+    private static Optional<ColumnType> convertDecimalOrNumeric(
+            Column column, DriverConfig<AbstractJdbcConfig> driverConfig) {
+        // 该字段的精度
+        int intValue = column.getPrecision().intValue();
+        if ((column.getType().contains("numeric") || column.getType().contains("decimal")) && intValue > 38) {
```

Comment:
This is redundant code:
`(column.getType().contains("numeric") || column.getType().contains("decimal")) &&`